python/ray/tune/trial_runner.py

```diff
@@ -112,7 +112,8 @@ def __init__(self,
         self._stop_queue = []
         self._metadata_checkpoint_dir = metadata_checkpoint_dir
 
-        self._session = datetime.today().strftime("%Y-%m-%d_%H-%M-%S")
+        self._start_time = datetime.today()
```

Comment:
datetime.now()?

Reply:
```
╰─ python -c 'from datetime import datetime; print(datetime.today())'
2019-03-03 15:26:22.770339
╭─ ~/Research/riselab/ray/python/ray/tune 
╰─ python -c 'from datetime import datetime; print(datetime.now())'
2019-03-03 15:26:26.618990
```
should be OK

Reply:
Oh this should be consistent.... good catch